cmd/mic/main.go

```diff
@@ -8,13 +8,14 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/client-go/rest"
```

Comment:
This repo has been following the import order of standard libraries -> local packages (`github.com/Azure/aad-pod-identity/...`) -> external packages. Let's keep this convention and add whitespace between those three import groups.

Reply:
@elsesiy  Thanks for addressing my comment. Could you also apply the same change to other files?

Reply:
Done! Just wondering why that matters since you could just always `goimports` the files and have it done the "right" way?

Reply:
Thanks for working on this. Linting tools, in general, are subjective and I don't think there is an absolute "right" way to lint a file. Although I agree with almost all of the changes in this PR, I think we should also follow some conventions laid down by the repository.

Reply:
Thanks for the explanation & I agree there's no single way to lint a file. I was just curious to understand why the repo chose this import ordering.